examples/conf-quickstart/druid/middleManager/runtime.properties

```diff
@@ -11,10 +11,6 @@ druid.indexer.task.baseTaskDir=var/druid/task
 # HTTP server threads
 druid.server.http.numThreads=9
 
-# Processing threads and buffers
-druid.processing.buffer.sizeBytes=256000000
-druid.processing.numThreads=2
```

Comment:
Reverted those example configs but with `druid.indexer.fork.property.` prefixes because IMO it's less confusing, leaves less room for mistakes (e. g. when configs for different node types are shared, or copied when people base configs for one node type on config for another node type, etc.)